### PR TITLE
chore: comment out custom form widget

### LINKED ISSUE #3599: Hide Custom Widget temporarily
See Chat here https://github.com/fossasia/open-event-frontend/issues/3598

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #3599

### CODE REVIEW

**Comment by iamareebjamal** [comment]
@prateekj117 Revire

**Comment by kushthedude** [comment]
Revire?

File: app/templates/components/forms/wizard/sessions-speakers-step.hbs [hunk 1]
```diff
@@ -316,7 +316,7 @@
   {{else}}
     <div class="spacer-50"></div>
   {{/if}}
-
+  {{!--  Commeting until next release for thorough testing and better UI.
```
Comment (prateekj117) [inline app/templates/components/forms/wizard/sessions-speakers-step.hbs]: Correct spelling of `Commenting`.

Comment (prateekj117) [reply app/templates/components/forms/wizard/sessions-speakers-step.hbs]: @iamareebjamal @kushthedude  
I think we should change the comment to: 
`Commented out until issue #3598 is solved and needs thorough testing`.

Comment (kushthedude) [reply app/templates/components/forms/wizard/sessions-speakers-step.hbs]: Hmm, sounds good.

**Comment by prateekj117** [comment]
@iamareebjamal LGTM.

**Pull request merged**